test/src/org/exist/xmldb/XmldbURITest.java

```diff
@@ -13,7 +13,7 @@
 
 public class XmldbURITest {
 
-    private final static String jettyPort = System.getProperty("jetty.port");
+    private final static String jettyPort = System.getProperty("jetty.port", "8080");
```

Comment:
I remember that @ljo added these system properties to be able to perform multiple tests in e.g. Jenkins. think of parallel  running tests. I am pretty sure this must be configurable, in one way or another